### PR TITLE
Feat : Added Cart Page

### LINKED ISSUE #15: Update: cart.html
Added links in header of cart.html

### PR DESCRIPTION
**Pull request by RahulSingh9131**
1. cart page
cartPage

### CODE REVIEW

File: css/screens/cart.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* css for cart page */
+
+/* css for header */
+header{
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
```
Comment (Gautam2305) [inline css/screens/cart.css:8]: i think these are default values and you don't need to mention them specifically here

**Review by Gautam2305 — COMMENTED** [review]
everything else looks good...... if you refactor code then you might want to add images from asset folder in you repo.....its not a much big problem...but if you want to add it...you can

File: css/screens/cart.css (hunk 1 above)
Comment (RahulSingh9131) [reply css/screens/cart.css:8]: Actually to overwrite the default style from component library for navigation, i added these lines.

**Review by rohit8686 — APPROVED** [review]
Everything looks good and naming is also good

**Pull request merged**